flow2ml/Data_Loader.py

```diff
@@ -80,6 +80,25 @@ def create_dataset(self,classPath):
 
               ind = self.classes.index(folderName)
               self.img_label[image_in_folder] = np.squeeze(np.eye(len(self.classes))[ind])
+    
+    if not any([i != -1 for i in [image.find("Images") for image in os.listdir(classPath)]]):
+
+      for image in list(os.listdir(classPath)):
```

Comment:
Please create a helper function for this copying part. And call that for necessary conditions. Because there is a lot of duplicate code.
Also add comments for the purpose of writing "if" condition.